app/presenters/brexit_taxons_presenter.rb

```diff
@@ -0,0 +1,41 @@
+class BrexitTaxonsPresenter
+  FEATURED_TAXONS = %w(
+    /going-and-being-abroad
+    /crime-justice-and-law
+    /transport
+    /environment
+    /business-and-industry
+    /education
+  ).freeze
+
+  def featured_taxons
+    @featured_taxons ||= FEATURED_TAXONS
+      .map { |base_path| Taxon.find(base_path) }
+      .map { |taxon| BrexitTaxonPresenter.new(taxon) }
+  end
+
+  def other_taxons
+    @other_taxons ||= ContentItem.find!('/')
+      .linked_items('level_one_taxons')
+      .reject { |content_item| FEATURED_TAXONS.include?(content_item.base_path) }
+      .select { |content_item| search_response_including_brexit(content_item.content_id, document_types).total.positive? }
+      .map { |content_item| Taxon.find(content_item.base_path) }
+      .map { |taxon| BrexitTaxonPresenter.new(taxon) }
+  end
+
+private
+
+  def search_response_including_brexit(content_id, filter_content_store_document_type)
+    params = {
+      count: 1,
+      fields: RummagerFields::TAXON_SEARCH_FIELDS,
```

Comment:
We could possibly use `link` or `title` or something instead of all the taxon fields, if we wanted to reduce returning payload to as small as it could be.  (I don't think we use them anywhere?)